src/basilisp/lang/util.py

```diff
@@ -14,6 +14,7 @@
 _DOUBLE_DOT_REPLACEMENT = "__DOT_DOT__"
 
 _MUNGE_REPLACEMENTS = {
+    "'": "__PRIME__",
```

Comment:
This was an unrelated change after PR #924 merged since it uses the `v'` nomenclature for a variable which was not previously covered by `munge`. I missed this change in #650.